docs/conf.py

```diff
@@ -313,3 +319,31 @@
 
 # If true, do not generate a @detailmenu in the "Top" node's menu.
 # texinfo_no_detailmenu = False
+
+link_files = {
+    f: dict(
+        using=dict(
+            GH='https://github.com',
+        ),
+        replace=[
+            dict(
+                pattern=r'((Issue|PR)\s?)?#(?P<issue_or_pr>\d+)',
+                url='{GH}/aio-libs/{project}/issues/{issue_or_pr}',
+            ),
+            dict(
+                pattern=r'^(?m)((?P<scm_version>v?\d+(\.\d+){1,2}))\n[-=]+\n',
+                with_scm='{text}\n{rev[timestamp]:%d %b %Y}\n',
+            ),
+            dict(
+                pattern=r'PEP[- ](?P<pep_number>\d+)',
```

Comment:
Better use [builtin markup](http://www.sphinx-doc.org/en/stable/markup/inline.html#role-pep).

Reply:
Again, what about viewing this on GitHub?

Reply:
No, it wouldn't - it's not a part of standard reST. But if want follow the way "rendered on GitHub like a candy" so you should drop sphinx in favour of plain markdown. But markdown is not for documentation (;

I think that rendering issue is not a problem since we have RTD which does everything right.